### ISSUE TITLE
Improve Gravatar alt-text

### ISSUE THREAD

**Issue by lcollette** [Automattic/wp-calypso#34800]
#### Steps to reproduce
1. Login to WordPress.com
2. Right-click to inspect gravatar
3. Note alt-text is "Me"

#### What I expected
The alt-text to be descriptive of the image, like "image of _username_".

#### What happened instead
The alt-text reads "Me". This lack clarity out of context.

#### Browser / OS version
All

#### Screenshot / Video

```<img alt="Me" class="gravatar" src="https://0.gravatar.com/avatar/fafc2374c06e05eb9a1217214cb1b25d?s=96&d=mm" width="18" height="18">```

**Comment by Aurorum** [comment]
I think the alt-text here is to suggest that clicking the image leads to the "Me section", as most other Gravtars use the username. Should it be something like "Leads to Me section" instead?

**Comment by sixhours** [comment]
It looks like, in the case of a link that contains only an image, the alt text should describe the link destination: https://dev.w3.org/html5/alt-techniques/developer.html#sec1

So I would suggest we say "My Profile" or "My Profile settings" since that's where the Gravatar in the masterbar goes.
